java/code/src/com/redhat/rhn/taskomatic/domain/TaskoSchedule.hbm.xml

```diff
@@ -77,15 +77,15 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
         </query>
 
         <query name="TaskoSchedule.listActiveByOrg">
-            <![CDATA[FROM com.redhat.rhn.taskomatic.domain.TaskoSchedule WHERE orgId = :org_id AND (activeFrom < :timestamp AND (activeTill is null OR :timestamp < activeTill))]]>
+            <![CDATA[FROM com.redhat.rhn.taskomatic.domain.TaskoSchedule WHERE orgId = :org_id AND (activeFrom < :timestamp AND (activeTill is null OR :timestamp < activeTill)) AND bunch.name NOT IN :filter]]>
```

Comment:
As for your question in the PR discussion: I think adjusting a hibernate queries is ok here. The only thing that rings a bell, is the situation, when `:filter` is an empty collection. Can you make sure, that the query would work in that case too? The query should be robust enough to handle that too.

Reply:
Right thanks for pointing that out. I should have tried that. Putting an empty Array in the query won't work.
I decided to adapt the method calling the query instead (using the `removeIf(...)` approach I used in the other case)